pf/internal/schemashim/type_schema.go

```diff
@@ -57,7 +58,7 @@ func (*typeSchema) Sensitive() bool { return false }
 
 func (s *typeSchema) Elem() interface{} {
 	switch tt := s.t.(type) {
-	case types.ObjectType:
+	case basetypes.ObjectTypable:
```

Comment:
Does this get rid of ObjectType entirely, in favor of the more flexible basetypes implementation?